src/components/menu/menu.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { ElMenu, ElMenuItemContainer, ElMenuItemGroup, ElMenuItemGroupTitle, Menu } from '.'
+import { Button } from '../button'
+
+const meta: Meta<typeof Menu> = {
+  title: 'Components/Menu',
+}
+
+export default meta
+type Story = StoryObj<typeof Menu>
+
+export const StylesOnlyUsage: StoryObj = {
+  render: () => {
+    return (
+      <ElMenu>
+        <ElMenuItemGroup>
+          <li>
+            <ElMenuItemGroupTitle>Group Title</ElMenuItemGroupTitle>
+          </li>
+          <ElMenuItemContainer>
+            <a href="/#">This is a Link</a>
+          </ElMenuItemContainer>
+          <ElMenuItemContainer>
+            <Button type="button">This is a Button</Button>
+          </ElMenuItemContainer>
+        </ElMenuItemGroup>
+      </ElMenu>
+    )
+  },
+}
+
```

Comment:
**suggestion:** I think is fine to show just the Menu.List usage in a story, but I think we should also have a complete story with a menu button that opens the menu when clicked etc.